docs/source/releases/v1_10_0_dev0.rst

```diff
@@ -3,184 +3,227 @@ Version 1.10.0.dev0 (2023-04-04)
 
 * Improve output from estimate area extent script
 * Add YAML-based plugin validator
-* Add "boundaries" plugin schema
-* Add "gridlines" plugin schema
-* Add "product_templates" plugin schema (for each of 11 families)
-* Convert old boundaries YAML to new format
-* Convert old gridlines YAML to new format
-* Convert old product_params YAML to new product_templates format
-* Update boundaries and gridlines schema to better handle defaults
-* Update plugin validator to follow references and add a base schema
+
```

Comment:
I finalized the release notes - please only update CHANGELOG.rst going forward.

Take a look at the final version of release notes to see if they have everything...

https://github.com/NRLMMD-GEOIPS/geoips/blob/bca10600e5fa896112c2dfaebf1e2c8ce71f948f/docs/source/releases/v1_10_0_dev0.rst

Reply:
Got it. Thanks for cleaning that up. These look good.